### PR TITLE
update favicons

### PR DESCRIPTION
**Pull request by Arhell**
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure

update favicons, fix issue https://github.com/istio/istio.io/issues/10185

### CODE REVIEW

**Comment by Arhell** [comment]
after fix https://realfavicongenerator.net/favicon_checker?protocol=https&site=deploy-preview-10187--preliminary-istio.netlify.app%2F#.YRhU_YgzaMo
& update fivicon.ico

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -88,7 +88,7 @@
         <link rel="alternate" type="application/rss+xml" title="Istio Blog and News" href="/feed.xml">
 
         <!-- Favicons: generated from img/istio-whitelogo-bluebackground-framed.svg by http://cthedot.de/icongen -->
-        <link rel="shortcut icon" href="/favicons/favicon.ico" >
+        <link rel="shortcut icon" href="/favicon.ico" >
```
Comment (craigbox) [inline layouts/_default/baseof.html:91]: Both seem to be the same file?

Comment (Arhell) [reply layouts/_default/baseof.html:91]: I deleted the extra file, there is the same
https://github.com/istio/istio.io/pull/10187/files#diff-662d8b2c61bb9159272d9b8f1864ebaf4bbb5cb83a30cde1c7afbba602d48db7
I updated it

Comment (craigbox) [reply layouts/_default/baseof.html:91]: Would it not be best to keep favicon.ico in /favicons ?  We can map a redirect for /favicon.ico if we need to.

Comment (Arhell) [reply layouts/_default/baseof.html:91]: Ok

Comment (craigbox) [reply layouts/_default/baseof.html:91]: try adding

```/favicon.ico /latest/favicons/favicon.ico 200```

to `layouts/index.redir` #46

**Comment by Arhell** [comment]
@craigbox  done

**Review by craigbox — APPROVED** [review]

**Pull request merged**